airgun/entities/host.py

```diff
@@ -378,3 +384,18 @@ class ShowRecommendations(NavigateStep):
     """Navigate to Insights recommendations page"""
 
     VIEW = CloudInsightsView
+
+
+@navigator.register(HostEntity, 'InsightsTab')
```

Comment:
This is also not required as this can be achieved using existing code

Reply:
There's one more way using which user can reach the insights tab on host details page.
User can click on recommendation count on all host page which navigates directly to insights tab.
I have added this code to test that navigation scenario.